openfe/setup/network.py

```diff
@@ -72,7 +72,7 @@ def enlarge_graph(self, *, edges=None, nodes=None) -> NetworkType:
         if nodes is None:
             nodes = set([])
 
-        return Network(self.edges | set(edges), self.nodes | set(nodes))
+        return Network(self._edges | set(edges), self._nodes | set(nodes))
```

Comment:
I'm 50/50 on this change. I usually try to write as much code as possible as if it was a client of the class and didn't have access to the underlying objects. That practice answers most of the complaints C++ people bring when they "prefer non-member non-friend functions." Technically, this does provide a slight speed-up, though. I'd be in favor if that was shown to be a real-world performance concern.

If this change is necessary because of changing other code to returning `iter(self.edges)`, I'd say this is another reason to just return the `frozenset` -- users might also want some functionality like this. (I can easily imagine a user wanted to check the overlap of two networks with `network1.nodes & network2.nodes`; why make them wrap in a set?)